app/models/report.rb

```diff
@@ -11,4 +11,12 @@ class Report < ApplicationRecord
   def statuses
     Status.where(id: status_ids)
   end
+
+  def media_attachments
```

Comment:
We might be able to turn `statuses` into a `has_many`, and then this could just be `has_many :media_attachments, through: :statuses`, instead of doing the looping this way.

If we can't do that, this can probably be shortened to something like `statuses.collect(&:media_attachments).flatten`, or similar.